package.json

```diff
@@ -21,8 +21,8 @@
   },
   "homepage": "https://github.com/svrcekmichal/react-sketchpad#readme",
   "dependencies": {
-    "react": "^0.14.0 || 15.0.0-0",
-    "react-dom": "^0.14.0 || 15.0.0-0",
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
```

Comment:
One more note, this will force users to have react 15.4.2+ to have installed, but lib is working with versions 0.14 and 15 of react. Can you fix this? It's last change before merge.